codalab/apps/api/views/worksheet_views.py

```diff
@@ -298,9 +298,9 @@ def post(self, request, uuid):
 
                 # Convert to ints
                 for key in ['request_cpus', 'request_gpus', 'request_priority']:
```

Comment:
Yup yup. I will make this change. I was unsure if this was used anywhere else in the code and I decided not to play with this earlier.